students/BRACH_Jakub/lesson_04_unit_tests/distance.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from math import sqrt, pow
+
+
+def distance(x1, y1, x2, y2):
+    return sqrt(pow((x2 - x1), 2) + pow((y2 - y1), 2))
+
+
+def test_TypeError_case():
+    with pytest.raises(TypeError):
+        distance()
+
+
+def test_ValueError_case():
+    with pytest.raises(TypeError):
+        distance("aoeu")
+
+
+def test_Zero_Length_case():
```

Comment:
strange function naming. 
why not make them all the same ?
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names